### PR TITLE
Add spdx slug to all files

### PR DESCRIPTION
**Pull request by Ainali**

### CODE REVIEW

File: logo-white.svg [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- SPDX-License-Identifier: CC0-1.0 -->
 <svg
    xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:cc="https://creativecommons.org/publicdomain/zero/1.0/"
```
Comment (Abbe98) [inline logo-white.svg:5]: XMLNS:* (XML Namespace decleration) is like a PREFIX decleration in SPARQL and does not effect the license. `http://creativecommons.org/ns#` is a vocabulary for various license related concepts, in this file the only used concept is: `http://creativecommons.org/ns#Work`. This change changes it to `https://creativecommons.org/publicdomain/zero/1.0/Work`.

**Comment by Abbe98** [comment]
Is this a common and documented pattern?

It works quite well right now with all files supporting XML comments but I worried it won't scale very well. Considering future formats not supporting XML comments and potential issues related to comments being retrained in production environments where files could be compressed, merged, etc. It might be easier to explicitly list everything not under CC0 in the README.

**Comment by Ainali** [comment]
> Is this a common and documented pattern?
> 
> It works quite well right now with all files supporting XML comments but I worried it won't scale very well. Considering future formats not supporting XML comments and potential issues related to comments being retrained in production environments where files could be compressed, merged, etc. It might be easier to explicitly list everything not under CC0 in the README.

Yes, spdx is common and documented. It can also take any sort of comments, as explained on: https://spdx.dev/ids/

It's [mandated for all public software in Italy](https://docs.italia.it/italia/developers-italia/gl-acquisition-and-reuse-software-for-pa-docs/en/stabile/software-reuse/open-licences-and-selecting-a-licence.html) and recommended by Standard for Public Code and many license compliance apps (like [REUSE](https://reuse.software) by FSFE) use it. The spdx specification is maintained by Linux Foundation.

**Comment by Abbe98** [comment]
SPDX-identifiers/the SPDX vocabulary I'm familiar with, however the comment practice is new to me.

It appears to be almost exclusively used within kernel development and I don't think it's a good fit for high-level development such as websites. Here are a few reasons that comes to mind:

 - Depending on file format the comment needs to be in particular locations (HTML is a great example)
 - It's not possible to ensure it's machine readable (comment syntax changes across files)
 - It only works in text files
 - Some formats doesn't support comments

**Comment by Ainali** [comment]
Let's close this one now and perhaps come back to a complete solution later.

**Comment by Abbe98** [comment]
FSFEs REUSE project appears to be aimed at addressing the exact issues I'm outlining above:

https://reuse.software/
https://reuse.software/spec/#dep5

**Comment by Ainali** [comment]
Yes. I suggest we keep this issue closed and create a new one for making the project REUSE compliant instead.

**Pull request closed without merging**